.ci/ci_cache_nemu.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Source to trap error line number
+source "${script_dir}/../lib/common.bash"
+
+WORKSPACE=${WORKSPACE:-$(pwd)}
+nemu_dir="/usr/local/bin/"
+
+cache_build_nemu() {
```

Comment:
I know you inherited/cloned  this script.... but....
with the name `cache_build_nemu`, I was expecting to see some sort of build command in here. Is it more like `cache_built_nemu` - that is, cache the one that has already been built??

If so, can I ask for a few minor changes (and maybe you can do the same to the other similar scripts at the same time in some other commits?):
- rename the function as above ;-)
- add a comment at the top of the file saying what it does
- and, does this script fail gracefully if `nemu` has not been built and is not in the place we expect to find it? If so, could you add some checks in here (and also maybe the other scripts? ;-) ) to do some sanity checks?

thx!
(btw, other than that, yes, caching `nemu` builds is a great idea to save CI time !!